src/i18n/config.json

```diff
@@ -50,7 +50,7 @@
     "name": "Persian",
     "langDir": "ltr",
     "dateFormat": "DD.MM.YYYY",
-    "hrefLang": "",
+    "hrefLang": "fr-FR",
     "enabled": false
```

Comment:
In order to test this language, change enabled to true 👀

Reply:
its done